app/routes/polls/components/PollDetail.tsx

```diff
@@ -30,28 +28,18 @@ const PollDetail = () => {
   );
 
   const poll = useAppSelector((state) => selectPollById(state, pollsId));
-  const fetching = useAppSelector((state) => state.polls.fetching);
   const actionGrant = poll?.actionGrant ?? [];
 
   if (!poll) {
```

Comment:
Yeah, I'm not sure this works. I've tried this on some pages, but they sometimes throw errors because somehow we get `poll=undefined` and `fetching=false` at the same time. Then it will try to render the page, which obviously doesn't work because there is no data.
The way I did it here is not great either though, as it will keep "loading" forever if the fetch fails.

Reply:
Hmmm well that's weird.. could it be an option to first do one if for the loading, and then a fallback under for the !poll? And have the fallback either returning null or some smol error? And add a comment as to why the fallback is needed?

Reply:
I figured out that it happens every time we load a page without SSR, because it renders the page once before starting to fetch the data. Seems like it's normally fast enough to not be visible though, so I can add an error-screen.